Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditor.cpp

```diff
@@ -952,6 +963,13 @@ namespace AzToolsFramework
         // refresh which Component Editor/Card looks selected in the Entity Outliner
         SetSelected(componentType == m_componentType);
     }
+
+    void ComponentEditor::ConnectPropertyChangeHandler(
```

Comment:
Is this class needed at all? Can we not create and connect a handler at the end of the ComponentEditor constructor?

Reply:
I'm keeping the prefab override code outside of ComponentEditor because ComponentEditor is also used in non-prefab related tools.